src/index.css

```diff
@@ -0,0 +1,187 @@
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-image: url(./images/cyptobg.jpeg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+body::before {
+  display: block;
+  content: '';
+  height: 110px;
+}
+
+code {
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    'Courier New',
+    monospace;
+}
+
+a {
+  text-decoration: none;
+}
+
+.autoimg {
+  padding: 9px 0 9px 0;
+}
+
+.header {
+  display: flex;
+  position: fixed;
+  top: 0;
+  background-color: #002855;
+  width: 100%;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.homepage,
+.center,
+.title {
+  color: #b1cb32;
+}
+
+.url {
+  color: #adb5bd;
+  text-decoration: underline;
+}
+
+.url:hover {
+  text-decoration: none;
+}
+
+.btn-con {
+  display: flex;
+  gap: 6px;
+  align-items: center;
+}
+
+.input {
+  display: flex;
+  justify-content: center;
+  margin: 15px 0 15px 0;
+}
+
+input {
+  width: 60%;
+  padding: 10px;
+  border: none;
+  border-radius: 10px;
+  font-size: 25px;
+  background-color: silver;
+  color: black;
+}
+
+.center {
+  text-align: center;
+}
+
+.break {
+  width: 50%;
+}
+
+.flex {
+  background-color: #001d3d;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  padding: 15px;
+  margin: 4px;
+  border-radius: 8px;
+}
+
+.main-container {
+  max-width: 90%;
+  margin: 0 auto;
+}
+
+.animate {
+  border: 10px solid #f3f3f3;
+  border-top: 10px solid #3498db;
+  position: relative;
+  border-radius: 50%;
+  margin: auto;
+  margin-bottom: 10px;
+  width: 80px;
+  height: 80px;
+  animation: spin 2s linear infinite;
+}
+
+@keyframes spin {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+}
+
```

Comment:
- [ ] One of the requirements is to adopt the layout composition of items used in the template. The layout shows a grid that has two items on a row whereas. In your design, we have one item per row. Kindly adjust this.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/211056950-0b844169-f5b8-4f45-a71b-95d391eef554.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/211057152-3507160a-1d5b-48b0-b249-9b2ae2e847f7.png)